server/src/main/java/org/apache/kafka/server/share/metrics/ShareGroupMetrics.java

```diff
@@ -82,7 +83,14 @@ public Histogram partitionLoadTimeMs() {
     @Override
     public void close() throws Exception {
         Arrays.stream(AcknowledgeType.values()).forEach(
-            m -> metricsGroup.removeMetric(RECORD_ACKNOWLEDGEMENTS_PER_SEC, Map.of(ACK_TYPE, m.toString())));
+            m -> metricsGroup.removeMetric(RECORD_ACKNOWLEDGEMENTS_PER_SEC, Map.of(ACK_TYPE_TAG, m.toString())));
         metricsGroup.removeMetric(PARTITION_LOAD_TIME_MS);
     }
+
+    private static String capitalize(String string) {
+        if (string == null || string.isEmpty()) {
+            return string;
+        }
+        return string.substring(0, 1).toUpperCase(Locale.ENGLISH) + string.substring(1);
```

Comment:
We tend to use `Locale.ROOT` in Kafka for upper/lower-casing strings. Indeed, `AcknowledgeType` uses `Locale.ROOT` so probably ought to be consistent.

Reply:
Done.